### PR TITLE
I changed the taste description of a few food items

### PR DESCRIPTION
**Pull request by PigeonLord**
What says on the tin. I deleted the taste descript of nutriments (since most foods contains nutriments, it made all food taste bland), changed "plant matter" to "vegetables" and changed the taste descript of soy sauce to  "soy".
🆑 Piccione
rscadd: Changed the taste message of a few food items
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/food.dm [hunk 1]
```diff
@@ -372,7 +371,7 @@
 	reagent_state = LIQUID
 	nutriment_factor = 5 * REAGENTS_METABOLISM
 	color = "#302000" // rgb: 48, 32, 0
-	taste_message = "ramen"
+	taste_message = "cheap ramen and memories"
```
Comment (taukausanake) [inline code/modules/reagents/chemistry/reagents/food.dm]: :+1: x :100:

**Comment by taukausanake** [comment]
I don't know about the plant matter being vegetables. They have a whole bunch of flavors. I rather make it say it tastes like grass or something else that denotes its leafy being

**Comment by PigeonLord** [comment]
@taukausanake The thing is, Pistachios and other non-leafy vegetables are "plant matter", so i think "vegetables" is more accurate

**Comment by DesolateG** [comment]
Soy sauce should be salty too in the taste description

File: code/modules/reagents/chemistry/reagents/food.dm (hunk 1 above)
Comment (DesolateG) [reply code/modules/reagents/chemistry/reagents/food.dm]: "Tastes like college hardships."

**Comment by tigercat2000** [comment]
:+1:

**Pull request merged**